### ISSUE TITLE
Improve contrast ratio of text in the Web UI

### ISSUE THREAD

**Issue by KopfKrieg** [duplicati/duplicati#4592]
- [x] I have searched open and closed issues for duplicates.
- [x] I have searched the [forum](https://forum.duplicati.com) for related topics.

----------------------------------------

## Environment info

- **Duplicati version**: 2.0.6.3
- **Operating system**: Windows 10 64bit
- **Backend**: Nextcloud/WebDAV

## Description

The contrast ratio in the Web UI is a bit low. It's 3.2:1 for the white background (#FFFFFF) to the grey text (#8F8F8F). For larger font sizes this might be okay, but for the current font size it's IMHO a bit too low.

[For good accessibility, at least a contrast ratio of 4.5:1 or 7:1 should be used on the body text](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast). In this case this would mean #777777 for the AA rating or #555555 for the AAA rating.

**Comment by samuel-w** [comment]
It seems that the grey text was chosen for readability in both light and dark mode, instead of having separate light/dark font colours.

**Comment by samuel-w** [comment]
I experimented with changing the contrast, and it looks kind of ugly. I used #555555 for the text colour.
![experiment](https://user-images.githubusercontent.com/67625312/130734757-c774fe06-3efc-49ce-ae5a-b269b7dcb0d5.PNG)
![experiment2](https://user-images.githubusercontent.com/67625312/130735041-c4b61c3d-61e8-4c6a-bee4-55baaa246420.PNG)
It does look a bit better in dark mode.
![edark](https://user-images.githubusercontent.com/67625312/130735444-7a557759-4380-4ec4-a98c-b918af5198b6.PNG)
Thoughts?
